packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -265,14 +288,20 @@ export default {
     margin: spacing(x2) 0;
 }
 
-.c-contactPreferences-labelText--disabled {
-    color: $color-content-disabled;
-}
-
 .c-contactPreferences-subtitle {
     @include font-size(heading-s);
 }
 
+.c-contactPreferences {
```

Comment:
Do you need to wrap these selectors in the outer `c-contactPreferences`?

Reply:
Yes it needs the higher specificity to override the style on `f-form-field`